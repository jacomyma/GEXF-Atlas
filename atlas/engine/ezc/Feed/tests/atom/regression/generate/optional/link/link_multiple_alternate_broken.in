<?php
return array( 'id' => 'Feed id',
              'title' => array( array( 'text' => 'Feed title',
                                       'type' => '' ) ),
              'updated' => 'Feed updated',
              'author' => array( array( 'name' => 'Author name' ) ),
              'link' => array( array( 'href' => 'Link href 1',
                                      'rel' => 'alternate',
                                      'type' => 'SAME TYPE',
                                      'hreflang' => 'SAME HREFLANG',
                                      'title' => 'Link title 1',
                                      'length' => 'Link length 1' ),
                               array( 'href' => 'Link href 2',
                                      'rel' => 'alternate',
                                      'type' => 'SAME TYPE',
                                      'hreflang' => 'SAME HREFLANG',
                                      'title' => 'Link title 2',
                                      'length' => 'Link length 2' ) ) );
?>
