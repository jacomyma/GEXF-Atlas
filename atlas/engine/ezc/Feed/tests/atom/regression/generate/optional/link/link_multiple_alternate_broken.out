The element '/feed/link@rel="alternate"' cannot appear more than once.
